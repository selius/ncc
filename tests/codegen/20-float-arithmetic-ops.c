int main()
{
	float a;
	float b;

	a = 4.5;
	b = 11.25;

	__print_float(a + b);
	__print_float(a - b);
	__print_float(a * b);
	__print_float(a / b);
	
	return 0;
}
