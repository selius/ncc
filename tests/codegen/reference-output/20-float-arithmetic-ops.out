15.750000
-6.750000
50.625000
0.400000
